bibblio.py

```diff
@@ -189,16 +189,23 @@ class BibblioCoverageProvider(WorkCoverageProvider):
     ]
 
     def __init__(self, _db, custom_list_identifier,
-                 api=None, catalogue_identifier=None, fiction=False):
+                 api=None, fiction=False, languages=None,
+                 catalogue_identifier=None, **kwargs):
         super(BibblioCoverageProvider, self).__init__(
             _db, self.SERVICE_NAME, self.OPERATION,
-            batch_size=self.DEFAULT_BATCH_SIZE
+            batch_size=self.DEFAULT_BATCH_SIZE, **kwargs
         )
 
         self.custom_list = CustomList.find(
-            self._db, DataSource.LIBRARY_STAFF, custom_list_identifier
+            self._db, DataSource.LIBRARY_STAFF, unicode(custom_list_identifier)
```

Comment:
This value might not be Unicode because in your script it comes in as a command-line argument, right? I think it would be better to convert it to Unicode right after pulling it off the command line.